### PR TITLE
[DONT Merge] - LBAC 9 Testing with EMRAPI

### PR DESCRIPTION
**Pull request by suthagar23**
Just added some Error based log lines. 

You need the updated appUI module including this [commit](https://github.com/openmrs/openmrs-module-appui/commit/992b9adfd3c7d7959b9c127f5c7553ab4d964fa7) to check this.

### CODE REVIEW

**Comment by dkayiwa** [comment]
In summary, what are these changes supposed to accomplish?

File: api/src/main/java/org/openmrs/module/locationbasedaccess/PatientSearchAdviser.java [hunk 1]
```diff
@@ -39,6 +40,9 @@ public boolean matches(Method method, Class targetClass) {
         else if (method.getName().equals("getPatient")) {
             return true;
         }
+        else if (method.getName().equals("getPatientByUuid")) {
```
Comment (dkayiwa) [inline api/src/main/java/org/openmrs/module/locationbasedaccess/PatientSearchAdviser.java]: Don't we already have this here? https://github.com/openmrs/openmrs-module-locationbasedaccess/blob/master/api/src/main/java/org/openmrs/module/locationbasedaccess/PatientSearchAdviser.java#L42

**Comment by suthagar23** [comment]
@dkayiwa  Actually this PR just contains some log lines which can help us to find the exact error point. Please follow this steps, 

1. Search for a patient and click to get the view in Patient Dashboard.
2. Then go to your terminal and mark the end line. (We need the logs after that line)
3. Just refresh the patient dashboard.
4. See the terminal for the log. You can see an exception which came from the EMRAPI module

**Comment by dkayiwa** [comment]
Can't i reproduce the problem without changes in this pull request?

**Comment by suthagar23** [comment]
Yes you can!!

**Comment by dkayiwa** [comment]
Can we then close this?
What are the steps to reproduce?

**Comment by suthagar23** [comment]
Yes. we can close this PR.

**Pull request closed without merging**